.travis.yml

```diff
@@ -4,33 +4,36 @@ language: php
 os:
   - linux
 dist: bionic
+
+addons:
+  apt:
+    packages:
+      - subversion
+
 services:
   - mysql
 
 php:
-  # TODO PHP 8.0
   - '7.4'
+  # TODO
+  #- '8.0'
 
 cache:
   directories:
     - '${HOME}/.composer/cache'
 
-# TODO maybe test building docs at some point
-
-addons:
-  apt:
-    packages:
-      - subversion
-
 before_install:
   - mysql -e 'CREATE DATABASE IF NOT EXISTS test;'
   - mysql -e 'GRANT ALL PRIVILEGES ON test.* TO test@localhost IDENTIFIED BY "test";'
+  - composer self-update --1
 
 install:
   - composer install --no-interaction --prefer-dist
   - DB_HOST=127.0.0.1 DB_NAME=test DB_USER=test DB_PASS=test bin/install-wp-tests.sh
 
 script:
-  - vendor/bin/phpunit
-  - ./vendor/bin/phpcs --standard=./phpcs.xml test/unit test/integration src
-  - vendor/bin/phpstan analyse
+  - composer run-script test
+  - composer run-script sniff
+  - composer run-script phpstan
+  # TODO
+  #- composer run-script docs:build
```

Comment:
This can be removed. I don't think we're going to have a full docs site for this plugin.

Reply:
Done.